Lib/test/test_builtin.py

```diff
@@ -1247,6 +1247,115 @@ def test_map_pickle(self):
             m2 = map(map_char, "Is this the real life?")
             self.check_iter_pickle(m1, list(m2), proto)
 
+    # def test_map_pickle_strict(self):
+    #     f2 = lambda x, y: (x, y)
+    #     a = (1, 2, 3)
+    #     b = (4, 5, 6)
+    #     t = [(1, 4), (2, 5), (3, 6)]
+    #     for proto in range(pickle.HIGHEST_PROTOCOL + 1):
+    #         m1 = map(f2, a, b, strict=True)
+    #         self.check_iter_pickle(m1, t, proto)
```

Comment:
How am I supposed to test this? Pickle can't save lambda's. And I wanted to keep the tests as much in line with zip() as possible.